rales_engine/spec/models/invoice_item_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe InvoiceItem, type: :model do
```

Comment:
be sure you test all your relationships on `InvoiceItem` and the custom methods you write within this class!